benchmark/tsconfig.json

```
@@ -0,0 +1,12 @@
+{
+    "compilerOptions": {
+        "target": "esnext",
+        "lib": ["esnext"],
+        // Dev types are JIT
+        "types": ["lua-types/jit"],
+        "moduleResolution": "node",
+        "strict": true,
+        "outDir": "dist",
+        "rootDir": "src"
```

To make it closer to https://github.com/TypeScriptToLua/TypeScriptToLua/blob/master/tsconfig.json
```suggestion
        "target": "esnext",
        "lib": ["esnext"],
        // Dev types are JIT
        "types": ["lua-types/jit"],
        "moduleResolution": "node",

        "outDir": "dist",
        "rootDir": "src",

        "strict": true,
        "noUnusedLocals": true,
        "noUnusedParameters": true
```